openshift/templates/patroni.yaml

```diff
@@ -95,7 +95,8 @@ objects:
       selector:
         matchLabels:
           statefulset: ${NAME}
-      serviceName: ${NAME}
+      service:
```

Comment:
@Sybrand apart from `batch/v1beta1` > `batch/v1`, this was the only other documented change that was relevant to us (`serviceName` > `service:name`)

Reply:
Eeek - I missed that one. Watch patroni carefully to make sure it takes!

Reply:
how would I check that?